components/progressTracker/ProgressStep.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import Box from '../box';
+import { TextSmall } from '../typography';
+
+class ProgressStep extends PureComponent {
+  render() {
+    const { label, active, completed, color } = this.props;
+    const classNames = cx(theme['progress-step'], theme[color], {
+      [theme['is-active']]: active,
+      [theme['is-completed']]: completed,
+    });
+    return (
+      <Box className={classNames}>
+        <TextSmall className={theme['progress-step__label']}>{label}</TextSmall>
+        <Box className={theme['status-bullet']} />
```

Comment:
Not really useful to make this a box. Just use a `span`.

Reply:
we also use a Box for the Bullet component, so I reused the same here methodology as we applied there to remain consistent.

Reply:
But you can't pass props to this Box component, so it's useless.